conf/puppet/hieradata/common.yaml

```diff
@@ -18,6 +18,7 @@ aem_curator::install_author::repository_volume_device: /dev/xvdb
 aem_curator::install_author::repository_volume_mount_point: /mnt/ebs1
 aem_curator::install_author::post_stop_sleep_secs: 300
 aem_curator::install_author::aem_license_base: "%{lookup('tmp_dir')}/license"
+aem_curator::install_author::aem_keystore_password: "%{lookup('AEM_Keystore_Password')}"
```

Comment:
This isn't related to hiera upgrade, please move to a separate feature branch / PR.